Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewViewModel.kt

```diff
@@ -0,0 +1,86 @@
+package de.rki.coronawarnapp.contactdiary.ui.overview
+
+import androidx.lifecycle.asLiveData
+import com.squareup.inject.assisted.AssistedInject
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.contactdiary.model.ContactDiaryLocationVisit
+import de.rki.coronawarnapp.contactdiary.model.ContactDiaryPersonEncounter
+import de.rki.coronawarnapp.contactdiary.storage.repo.ContactDiaryRepository
+import de.rki.coronawarnapp.contactdiary.ui.overview.adapter.ListItem
+import de.rki.coronawarnapp.ui.SingleLiveEvent
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModel
+import de.rki.coronawarnapp.util.viewmodel.SimpleCWAViewModelFactory
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.flowOf
+import org.joda.time.LocalDate
+import timber.log.Timber
+
+class ContactDiaryOverviewViewModel @AssistedInject constructor(
+    contactDiaryRepository: ContactDiaryRepository
+) : CWAViewModel() {
+
+    val routeToScreen: SingleLiveEvent<ContactDiaryOverviewNavigationEvents> = SingleLiveEvent()
+
+    private val dates = flowOf((0..13).map { LocalDate.now().minusDays(it) })
```

Comment:
minor: introduce a const val for the total number of days

Reply:
done